examples/README.md

```diff
@@ -1,22 +1,93 @@
-# Contents of this Document
+# Examples showcasing TorchServe Features and Integrations
+
+## TorchServe Internals
+
 * [Creating mar file for an eager mode model](#creating-mar-file-for-eager-mode-model)
 * [Creating mar file for torchscript mode model](#creating-mar-file-for-torchscript-mode-model)
-* [Serving torchvision image classification models](#serving-image-classification-models)
 * [Serving custom model with custom service handler](#serving-custom-model-with-custom-service-handler)
-* [Serving text classification model](#serving-text-classification-model)
-* [Serving text classification model with scriptable tokenizer](#serving-text-classification-model-with-scriptable-tokenizer)
+* [Creating a Workflow](#serving-image-classification-workflow)
+* [Custom Metrics](custom_metrics)
+* [Dynamic Batch Processing](image_classifier/resnet_152_batch)
+* [Dynamic Batched Async Requests](#serving-image-classifier-model-for-on-premise-near-real-time-video)
+
+## TorchServe Integrations
```

Comment:
I don't think the placement for the logos is consistent, sometimes it shows on the same line, sometimes below. Also I believe some logos are missing like the TensorRT one

Also we should update `docs/README.md` since that's what renders on the page

Last thing is also make sure the 2 lint jobs pass for links and spellchecking

Reply:
My html skills are rudimentary. I've been trying to play around with the html syntax to make these consistent. Will try my best to fix this

Reply:
Done